src/main/java/org/sagebionetworks/web/client/widget/entity/PreviewWidgetView.java

```diff
@@ -14,6 +16,7 @@
 	public void setImagePreview(String fullFileUrl, String previewUrl);
 	public void setCodePreview(String text);
 	public void setTextPreview(String text);
+	public void setVideoPreview(VideoWidget vw);
```

Comment:
Please change to a generic setPreviewWidget(Widget w); (and do the videoWidget.asWidget() in the presenter)